### ISSUE TITLE
Packaging of Windows Aplication - .zip / Installer

### ISSUE THREAD

**Issue by AndreasFreund** [amosproj/amos2024ss07-updating-flash-boot-loader#23]
## User story
1. As a user
2. I want a simple setup / extraction process
3. So that I can run the application

## Acceptance criteria
* Create bash file to create .zip file or installer containing all neccessary parts for the application
* Test on other PC (e.g. to catch .dll files which were required but not included in .zip file)
* Don't include Vector .dll files, just Qt. User needs to install Vector driver themselves
* Split Build process into build, run, package

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
